src/App.tsx

```diff
@@ -1,109 +1,24 @@
 import React from 'react';
-
+import { Post } from './react-app-env';
+import { PostList } from './components/PostList';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+const posts: Post[] | [] = postsFromServer.map(post => ({
+  ...post,
+  user: usersFromServer.find(user => user.id === post.userId) || null,
+  comments: commentsFromServer
+    .filter(comment => comment.postId === post.id) || [],
```

Comment:
filter method returns empty array by default if it's not found any elements, || [] here is redundant